### PR TITLE
Security Uniform Tweaks

### PR DESCRIPTION
**Pull request by alsoandanswer**
One of the main qualms about the new PMCG uniforms is the grey color. So I've made it blacker.
dreammaker_ADvl9CmpPy

Here's some comparisons to the old uniforms to the new ones. I've tried to keep them as close to the originals as possible while implementing my new shading
dreammaker_AxoeQGs9hL
dreammaker_jqBAGqVbNn

Here's some alternate uniform selections as well. Alongside EPMC getting a touch up, Zavod gets their "old" alternate uniforms back, and I've newly added Idris selections too. I hope you like them.
dreammaker_eRhgYCih4M
New softcap selections for Idris and Necrosec. Tweaks the EPMC cap to match the black color used in their uniforms.
I decided it was fine to make the EPMC cap black, since the PMCG cap is already grey to avoid redundancy.
dreammaker_v0Lhils6ji

note : this images have not been updated, please look at the .dmi for the latest version

Feedback thread : https://forums.aurorastation.org/topic/17245-security-uniform-tweaks-again/#comment-157051

### CODE REVIEW

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Don't use necropolis in pathing - use Zavodskoi to be consistent with 1) current naming of the corporation and 2) other paths. Look at the forum thread for my sprite feedback.

The base type `/obj/item/clothing/under/rank/security` does not have a worn sprite. Is this intended?

**Comment by alsoandanswer** [comment]
!review

**Comment by anconfuzedrock** [comment]
from loadout, the black zavod cap + beret are both options but they seem to be invisible.

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 1]
```diff
@@ -155,48 +195,44 @@
...
-	var/list/epmc_headwear = list()
-	epmc_headwear["beret, EPMC"] = /obj/item/clothing/head/beret/corporate/pmc/epmc
-	epmc_headwear["softcap, EPMC"] = /obj/item/clothing/head/softcap/pmc/epmc
-	gear_tweaks += new /datum/gear_tweak/path(epmc_headwear)
-
-/datum/gear/faction/epmc_sec_uniforms
+/datum/gear/faction/pmcg_sec_uniforms
 	display_name = "EPMC security uniform selection"
-	description = "A selection of EPMC security uniforms."
+	description = "A selection of PCMG and EPMC security uniforms."
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:224]:
```suggestion
	description = "A selection of PMCG and EPMC security uniforms."
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 2]
```diff
@@ -155,48 +195,44 @@
 	pmc_labcoats["labcoat alt, PMCG"] = /obj/item/clothing/suit/storage/toggle/labcoat/pmc/alt
 	gear_tweaks += new /datum/gear_tweak/path(pmc_labcoats)
 
-/datum/gear/faction/pmc_patch
-	display_name = "PMCG armband"
-	path = /obj/item/clothing/accessory/armband/pmc
-	slot = slot_tie
+/datum/gear/faction/pcmg_headwear
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:198]:
```suggestion
/datum/gear/faction/pmcg_headwear
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 3]
```diff
@@ -155,48 +195,44 @@
...
+	display_name = "PMCG and EPMC headwear selection"
+	description = "A selection of PCMG and EPMC headwear."
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:200]:
```suggestion
	description = "A selection of PMCG and EPMC headwear."
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 4]
```diff
@@ -155,48 +195,44 @@
...
+	path = /obj/item/clothing/head/softcap/pmc
+	slot = slot_head
 	faction = "Private Military Contracting Group"
 
+/datum/gear/faction/pcmg_headwear/New()
+	..()
+	var/list/pcmg_headwear = list()
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:207]:
```suggestion
	var/list/pmcg_headwear = list()
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 5]
```diff
@@ -155,48 +195,44 @@
...
+	pcmg_headwear["PCMG beret"] = /obj/item/clothing/head/beret/corporate/pmc
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:208]:
```suggestion
	pcmg_headwear["PMCG beret"] = /obj/item/clothing/head/beret/corporate/pmc
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 6]
```diff
@@ -155,48 +195,44 @@
...
+	pcmg_headwear["PCMG softcap"] = /obj/item/clothing/head/softcap/pmc
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:209]:
```suggestion
	pcmg_headwear["PMCG softcap"] = /obj/item/clothing/head/softcap/pmc
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 7]
```diff
@@ -155,48 +195,44 @@
...
+	pcmg_headwear["PCMG softcap, alt"] = /obj/item/clothing/head/softcap/pmc/alt
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:210]:
```suggestion
	pcmg_headwear["PMCG softcap, alt"] = /obj/item/clothing/head/softcap/pmc/alt
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 8]
```diff
@@ -155,48 +195,44 @@
...
+	pcmg_headwear["EPMC beret"] = /obj/item/clothing/head/beret/corporate/pmc/epmc
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:211]:
```suggestion
	pmcg_headwear["EPMC beret"] = /obj/item/clothing/head/beret/corporate/pmc/epmc
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 9]
```diff
@@ -155,48 +195,44 @@
...
+	pcmg_headwear["EPMC softcap"] = /obj/item/clothing/head/softcap/pmc/epmc
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:212]:
```suggestion
	pmcg_headwear["EPMC softcap"] = /obj/item/clothing/head/softcap/pmc/epmc
```

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 10]
```diff
@@ -155,48 +195,44 @@
...
+	gear_tweaks += new /datum/gear_tweak/path(pcmg_headwear)
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:213]:
```suggestion
	gear_tweaks += new /datum/gear_tweak/path(pmcg_headwear)
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**